docs/resources/job.md

```diff
@@ -3,63 +3,45 @@
 page_title: "qovery_job Resource - terraform-provider-qovery"
 subcategory: ""
 description: |-
-  Provides a Qovery job resource. This can be used to create and manage Qovery job registry.
+  Provides a Qovery job resource. This can be used to create and manage Qovery job registry. Cron job example https://github.com/Qovery/terraform-examples/tree/main/examples/deploy-a-cron-job Lifecycle job example https://github.com/Qovery/terraform-examples/tree/main/examples/deploy-a-lifecycle-job
 ---
 
 # qovery_job (Resource)
 
-Provides a Qovery job resource. This can be used to create and manage Qovery job registry.
+Provides a Qovery job resource. This can be used to create and manage Qovery job registry. [Cron job example](https://github.com/Qovery/terraform-examples/tree/main/examples/deploy-a-cron-job) [Lifecycle job example](https://github.com/Qovery/terraform-examples/tree/main/examples/deploy-a-lifecycle-job)
 
 ## Example Usage
 
 ```terraform
 resource "qovery_job" "my_job" {
-  # Required
-  environment_id = qovery_environment.my_environment.id
-  name           = "test-job"
-
-  # Optional
-  auto_preview         = "true"
```

Comment:
true that these are optional but in 99% of the case you will configure them so let's keep them in the example? 

auto_preview
CPU
memory
max_duration_seconds
max_nb_restart
environment_variables

Reply:
Gonne put them back